### PR TITLE
Move CI to GitHub Actions

### PR DESCRIPTION
**Pull request by stevesoltys**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,33 +0,0 @@
-dist: trusty
-
-jdk:
-  - openjdk11
-
-language: android
-android:
-  components:
-    - build-tools-30.0.2
-    - android-30
```
Comment (chirayudesai) [inline .travis.yml:10]: Don't need to set this either?

Does this figure out the versions out automatically somehow based on gradle config?

Comment (stevesoltys) [reply .travis.yml:10]: This was to tell Travis what Android version to install on the build image. I think all versions of the Android SDK are installed on the GitHub build image.

This seems to be what determines which Android version the build uses: https://github.com/seedvault-app/seedvault/blob/master/app/build.gradle#L20

**Review by chirayudesai — DISMISSED** [review]
Just gotta disable "Travis CI - Pull Request Expected — Waiting for status to be reported
Required" then

File: .travis.yml (hunk 1 above)
Comment (chirayudesai) [reply .travis.yml:10]: Sweet

**Review by grote — CHANGES_REQUESTED** [review]
We should also change/remove this:
https://github.com/seedvault-app/seedvault/blob/7694eb3cf9052732878715eb1acc89957bf8da35/app/build.gradle#L21

Also, will the travis build badge still work after this?

**Review by chirayudesai — APPROVED** [review]

**Review by grote — APPROVED** [review]

**Pull request merged**